container/view.go

```diff
@@ -416,6 +417,10 @@ func (v *View) transform(ctr *Container) *Snapshot {
 	}
 	snapshot.NetworkSettings = &container.NetworkSettingsSummary{Networks: networks}
 
+	if ctr.ImageManifest != nil && ctr.ImageManifest.Platform == nil {
+		ctr.ImageManifest.Platform = &ctr.ImagePlatform
```

Comment:
`ctr` is supposed to be immutable, we shouldn't patch it in `ctr` directly.

This should be patched in `snapshot.Summary`

Reply:
cc @jsternberg